code/game/machinery/computer/extraction_cargo.dm

```diff
@@ -25,10 +25,16 @@
 		new /datum/data/extraction_cargo("Drain Monitor ",				/obj/item/powered_gadget/detector_gadget/abnormality,				200, CAT_GADGET) = 1,
 		new /datum/data/extraction_cargo("Keen-Sense Rangefinder ",		/obj/item/powered_gadget/detector_gadget/ordeal,					200, CAT_GADGET) = 1,
 		new /datum/data/extraction_cargo("EMAIS	Capacity Upgrade ",		/obj/item/hypo_upgrade/cap_increase,								200, CAT_GADGET) = 1,
+		new /datum/data/extraction_cargo("Instant Clerkbot Constructor ",/obj/item/powered_gadget/clerkbot_gadget,							250, CAT_GADGET) = 1,
+		new /datum/data/extraction_cargo("C-Fear Protection Injector ",	/obj/item/lc13_clerk_fear_immunity_injector,						300, CAT_GADGET) = 1,
```

Comment:
```suggestion
```
For now. Want it reworked before being put in.

Reply:
commented out